dropwizard-testing/src/main/java/io/dropwizard/testing/junit/ResourceTestRule.java

```diff
@@ -45,6 +32,9 @@
         BootstrapLogging.bootstrap();
     }
 
+    /**
+     * A builder for {@link ResourceTestRule} that allows to configure a Jersey testing environment.
```

Comment:
How about "A {@link ResourceTestRule} builder which enables configuration of a Jersey testing environment."?